x-pack/plugins/stack_connectors/public/connector_types/slack_api/slack_connectors.tsx

```diff
@@ -33,15 +42,123 @@ const getSecretsFormSchema = (docLinks: DocLinksStart): SecretsFieldSchema[] =>
   },
 ];
 
+const getConfigFormSchemaAfterSecrets = (
+  options: EuiComboBoxOptionOption[],
+  isLoading: boolean,
+  isDisabled: boolean
+): ConfigFieldSchema[] => [
+  {
+    id: 'allowedChannels',
+    isRequired: false,
+    label: i18n.ALLOWED_CHANNELS,
+    helpText: (
+      <FormattedMessage
+        id="xpack.stackConnectors.components.slack_api.allowedChannelsText"
+        defaultMessage="It will only fetch channels with a valid Slack Web API token. If it is empty, we will allow all channels."
```

Comment:
Does this mean there's an additional channel-level token or app scope required? Or is this just meant to describe the default behaviour? If the latter:
```suggestion
        defaultMessage="By default, the connector can access all channels within the scope of the Slack app."
```

Reply:
just meant to describe the default behavior

Reply:
Cool, thanks. I added the "within the scope of the Slack app" bit to allude to the "Add scope" step in https://www.elastic.co/guide/en/kibana/master/slack-action-type.html#configuring-slack-web-api. If you want to omit that, however, it could just be:

```suggestion
        defaultMessage="By default, the connector can access all channels."
```

Since there's extra work required for that to be true, I'd vote for mentioning the scope.